packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -784,23 +788,22 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     } = this.props;
 
     let id = this._id;
-
+    // Render using regular mapping
     return (
       <div
         id={ id + '-list' }
         className={ this._classNames.optionsContainer }
         aria-labelledby={ id + '-label' }
         role='listbox'
       >
-        { this.state.currentOptions.map((item, index) => onRenderItem({ ...item, index } as ISelectableOption, this._onRenderItem)) }
+        { props.options.map((item) => onRenderItem(item, this._onRenderItem)) }
```

Comment:
Is this change needed anymore?

Reply:
Leaked from the other change, thanks. Will remove